### PR TITLE
[Frontend] fix CVE and add frontend test

### PR DESCRIPTION
**Pull request by lanking520**
## Description ##

ejs dpendencies is causing critical CVE and patch with latest

### CODE REVIEW

**Comment by lanking520** [comment]
@c007456

File: .github/workflows/frontend.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      - uses: actions/cache@v2
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+      - name: run npm build
+        run: |
+          cd plugins/management-console
+          ./gradlew bCA
```
Comment (frankfliu) [inline .github/workflows/frontend.yml:32]: This is part of gradle build already, we don't need a separate workflow

Comment (lanking520) [reply .github/workflows/frontend.yml:32]: How do you run this without setting up npm?

Comment (frankfliu) [reply .github/workflows/frontend.yml:32]: gradle node plugin download the node and npm

**Review by frankfliu — APPROVED** [review]

**Pull request merged**